docs/config/aws_template.md

```diff
@@ -0,0 +1,441 @@
+# Deploying to AWS
+
+!!! danger
```

Comment:
Danger!!! - Should there be a warning here?

Reply:
Yeah, 'danger' might be too harsh. I want it to be a warning that stands out a bit from other warnings. Changed it to accomplish that.

Reply:
<img width="1051" alt="image" src="https://github.com/tsdat/tsdat/assets/24307537/9eddb96b-4d96-4eb7-b1f9-595bb16947db">